cmd/fleetctl/query_test.go

```diff
@@ -299,7 +300,15 @@ func TestAdHocLiveQuery(t *testing.T) {
 		)
 	}()
 
+	// test label not found
+	_, err = runAppNoChecks([]string{"query", "--hosts", "1234", "--labels", "iamnotalabel", "--query", "select 42, * from time"})
+	assert.ErrorContains(t, err, "Invalid label name(s): iamnotalabel.")
+
+	// test if some labels were not found
+	_, err = runAppNoChecks([]string{"query", "--labels", "label1, mac, windows", "--hosts", "1234", "--query", "select 42, * from time"})
+	assert.ErrorContains(t, err, "Invalid label name(s): mac, windows.")
+
 	expected := `{"host":"somehostname","rows":[{"bing":"fds","host_display_name":"somehostname","host_hostname":"somehostname"}]}
 `
-	assert.Equal(t, expected, runAppForTest(t, []string{"query", "--hosts", "1234", "--query", "select 42, * from time"}))
+	assert.Equal(t, expected, runAppForTest(t, []string{"query", "--labels", "", "--hosts", "1234", "--query", "select 42, * from time"}))
```

Comment:
Does this test fail if you don't add `"--labels", "", `?

Reply:
This test passes without that param. 
```
13:53:26 [ksykulev:~/projects/fleet] (23015-labels-param-validation) !↑ $ NETWORK_TEST=1 MYSQL_TEST=1 REDIS_TEST=1 SAML_IDP_TEST=1 go test github.com/fleetdm/fleet/v4/cmd/fleetctl -v -run=TestAdHocLiveQuery -race=true -parallel=8 -coverprofile=coverage.txt -covermode=atomic -failfast -tags full
# github.com/fleetdm/fleet/v4/cmd/fleetctl.test
ld: warning: '/private/var/folders/6r/v47y77pn7tz9bvm4ct98g6yw0000gn/T/go-link-1158758029/000035.o' has malformed LC_DYSYMTAB, expected 98 undefined symbols to start at index 1626, found 95 undefined symbols starting at index 1626
=== RUN   TestAdHocLiveQuery
{"level":"error","method":"POST","took":"1.446458ms","uri":"/api/latest/fleet/queries/run_by_identifiers","user":"admin1@example.com","uuid":"01c9b927-4275-4e2e-9866-65ee0efd8c9f"}
{"level":"error","method":"POST","took":"785.083µs","uri":"/api/latest/fleet/queries/run_by_identifiers","user":"admin1@example.com","uuid":"566cff3a-5336-4997-95ff-bb005fa6a725"}
{"level":"debug","method":"POST","numHosts":1,"query_id":null,"sql":"select 42, * from time","took":"4.827ms","uri":"/api/latest/fleet/queries/run_by_identifiers","user":"admin1@example.com"}
 100% responded (100% online) | 1/1 targeted hosts (1/1 online)
--- PASS: TestAdHocLiveQuery (7.08s)
PASS
coverage: 7.9% of statements
ok  	github.com/fleetdm/fleet/v4/cmd/fleetctl	8.766s	coverage: 7.9% of statements
13:54:11 [ksykulev:~/projects/fleet] (23015-labels-param-validation) !↑ $ git diff
diff --git a/cmd/fleetctl/query_test.go b/cmd/fleetctl/query_test.go
index c862b6bbe..8e5441b58 100644
--- a/cmd/fleetctl/query_test.go
+++ b/cmd/fleetctl/query_test.go
@@ -310,5 +310,5 @@ func TestAdHocLiveQuery(t *testing.T) {
 
        expected := `{"host":"somehostname","rows":[{"bing":"fds","host_display_name":"somehostname","host_hostname":"somehostname"}]}
 `
-       assert.Equal(t, expected, runAppForTest(t, []string{"query", "--labels", "", "--hosts", "1234", "--query", "select 42, * from time"}))
+       assert.Equal(t, expected, runAppForTest(t, []string{"query", "--hosts", "1234", "--query", "select 42, * from time"}))
 }
```

The intent was to test it with a blank labels flag. But I should really pull that out into a separate test, like I did the ones above, rather than this. I will make the change.

Reply:
remove `--label, ""` instead to avoid confusion